com.valvesoftware.Steam.yml

```
@@ -63,6 +63,10 @@ add-extensions:
     download-if: active-gl-driver
     enable-if: active-gl-driver
 
+  com.valvesoftware.Steam:
```

I guess since this extension point is specific to Proton, it should be named accordingly.
```suggestion
  com.valvesoftware.Steam.Proton:
```
And proton build can be named like `com.valvesoftware.Steam.Proton.tkg`